pkg/gatherers/clusterconfig/events_filtering_test.go

```diff
@@ -0,0 +1,77 @@
+package clusterconfig
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_filterEvents_LastTimeStamp(t *testing.T) {
+	oldEvent := v1.Event{
+		ObjectMeta:    metav1.ObjectMeta{Name: "oldEvent"},
+		LastTimestamp: metav1.Time{},
+		Type:          "Normal",
+	}
+	newEvent := v1.Event{
+		ObjectMeta:    metav1.ObjectMeta{Name: "newEvent"},
+		LastTimestamp: metav1.Now(),
+		Type:          "Normal",
+	}
+
+	events := v1.EventList{}
+	events.Items = append(events.Items, newEvent, oldEvent)
+
+	filteredEvents := filterEvents(time.Duration(1*time.Minute), &events, "")
+
+	if !reflect.DeepEqual(filteredEvents.Items[0], newEvent) {
+		t.Errorf("filterEvents() = %v, want %v", filteredEvents.Items[0], newEvent)
+	}
+}
+
+func Test_filterEvents_Warning(t *testing.T) {
+	normalEvent := v1.Event{
+		ObjectMeta:    metav1.ObjectMeta{Name: "normalEvent"},
+		LastTimestamp: metav1.Now(),
+		Type:          "Normal",
+	}
+	warningEvent := v1.Event{
+		ObjectMeta:    metav1.ObjectMeta{Name: "warningEvent"},
+		LastTimestamp: metav1.Now(),
+		Type:          "Warning",
+	}
```

Comment:
This test and the one above could be merged into one (See e.g [this](https://github.com/openshift/insights-operator/blob/master/pkg/gatherers/clusterconfig/namespaces_with_overlapping_uids_test.go#L15) for how you can define multiple similar tests.
I would also recommend to test more than one event with the "abnormal" event type :)